pennylane/transforms/metric_tensor.py

```diff
@@ -315,7 +315,6 @@ def circuit(weights):
             "as trainable."
         )
 
-    # pylint: disable=too-many-arguments

```

Comment:
It seems that this (duplicate) `disable` line actually blocked the other `disable` line from doing its job. Didn't know that this is the case...